kolena/_experimental/object_detection/dataset.py

```diff
@@ -240,10 +242,20 @@ def _compute_metrics(
     for record in pred_df.itertuples():
         ground_truths = record[idx[ground_truth]]
         inferences = record[idx[inference]]
+        ignored_ground_truths = [
+            gt
+            for gt in ground_truths
+            if ignore_gt_property is not None
+            and hasattr(gt, ignore_gt_property)
+            and isinstance(getattr(gt, ignore_gt_property), bool)
+            and getattr(gt, ignore_gt_property)
+        ]
+        unignored_ground_truths = [gt for gt in ground_truths if gt not in ignored_ground_truths]
```

Comment:
I'm not sure this is quite what we're targeting here -- my read is that ignored bboxes should still be matched, but not surface as FNs if unmatched. Maybe something we should clarify before committing to.

Reply:
That's the behavior that this accomplishes. `_match_inferences_single_class_pascal_voc` [matches against](https://github.com/kolenaIO/kolena/blob/ea38bdce1e71facc3bf783d5a14b6bc85ad9dc32/kolena/metrics/_geometry.py#L147) `ground_truths + ignored_ground_truths`, but [doesn't include](https://github.com/kolenaIO/kolena/blob/ea38bdce1e71facc3bf783d5a14b6bc85ad9dc32/kolena/metrics/_geometry.py#L176) `ignored_ground_truths` in the FN and doesn't include any of the inferences that match with anything in `ignored_ground_truths` in the FP.

Reply:
This is also verified in the unit test

Reply:
Ah, great (thank you)

Reply:
Ah, great (thank you)